src/python/zquantum/core/interfaces/estimator.py

```diff
@@ -40,8 +43,9 @@ def get_estimated_expectation_values(
         """
         raise NotImplementedError
 
+    @staticmethod
```

Comment:
We changed it to a staticmethod because it does not use `self` now, which is good (I would rather extract it to a module-level function, but that's matter of taste). However, when you see how this is called, the `estimator_name` is always set to `type(self).__name__`. What do you think about making it an instance method again but instead of passing `estimator_name`, just use `self` to obtain the name?

Reply:
Thank you for the comments! I agree with your points :)  

What do you think about just getting rid of this function? I kind of regret adding it and think its sort of overkill as an `Estimator`-specific helper. Maybe in the future, we can have a `logging_util` module which checks if something is `None` and logs it. Something like:

```
# logging_util.py
def ignore_if_not_none(logger, parameter_name, parameter_value)
```

 If you think it makes sense to have, then I'd be happy to move this to a module-level function before this is merged.

Reply:
I agree! So let's just delete it, and we are good to go! :fire: